dev/tools/localization/language_subtag_registry.dart

```diff
@@ -46629,7 +46629,7 @@ Prefix: de
 Type: variant
 Subtag: abl1943
 Description: Orthographic formulation of 1943 - Official in Brazil
-  (Formulário Ortográfico de 1943 - Oficial no Brasil)
```

Comment:
While no native speaker of the language, I believe "Oficial" was correct here. Can you revert this?

Reply:
done sir !